linode/manager
# M3 4608 test updates for ci runs

**How to verify that that the test works:**
- Make sure mock service worker is off.
- yarn up
- yarn cy:e2e **(runs headless)**
- Result should be "✔  All specs passed!"

Make sure your .env contains:
REACT_APP_LOGIN_ROOT='https://login.linode.com'
REACT_APP_API_ROOT='https://api.linode.com/v4'
REACT_APP_APP_ROOT='http://localhost:3000'
REACT_APP_LAUNCH_DARKLY_ID=(CMR flag)
REACT_APP_CLIENT_ID=(https://cloud.linode.com/profile/clients, copy ID)
MANAGER_OAUTH=(create in https://cloud.linode.com/profile/tokens > "Add a Person Access Token", Save the token that appears in the modal, you will only see it once)
